client/landing/gutenboarding/onboarding-block/plans/index.tsx

```diff
@@ -102,15 +102,23 @@ const PlansStep: React.FunctionComponent< Props > = ( { isModal } ) => {
 			<div>
 				<Title>{ __( 'Select a plan' ) }</Title>
 				<SubTitle>
-					{ sprintf(
-						/* translators: number of days */
-						__(
-							'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within %1$d days.'
-						),
-						// Only show a 7-days refund window if billing period is
-						// defined AND is set to monthl (its value starts as undefined)
-						billingPeriod === 'MONTHLY' ? 7 : 14
-					) }
+					{ billingPeriod === 'MONTHLY'
+						? sprintf(
+								/* translators: %1$d is number of days */
+								__(
+									'Pick a plan that’s right for you. There is no risk, you can cancel your monthly plan for a full refund within %1$d days.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within %1$d days.' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/monthly-plans-gutenboarding-copy-changes)